src/std/swank/autodoc.ss

```diff
@@ -0,0 +1,93 @@
+(import :gerbil/expander :std/swank/message :std/sugar
+	:std/swank/context :std/swank/api :std/format
+	:std/srfi/13)
+(export #t)
+
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;; Completions, Autodoc and Signature
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+
+;; search for a procedure in 0gambit-procedures
+;; returns the procedure symbol if it finds it
+
+(gxc#optimizer-info-init!)
```

Comment:
you also need to import `:gerbil/compiler` to make sure this exists.

Reply:
I would actually import `:gerbil/compiler/optimize-base` to avoid having to `gxc#` stuff.